src/Chat/Plugin/CommandOnlyPlugin.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Room11\Jeeves\Chat\Plugin;
+
+use Room11\Jeeves\Chat\Command\Message;
+
+/**
+ * Provides default interface method implementations for plugins which only handle commands
+ */
+trait CommandOnlyPlugin
+{
+    public function handleMessage(Message $message): \Generator { yield; }
```

Comment:
Actually, because it would never be called, it could still just be an empty method body in this case (https://3v4l.org/sbh6J),